package.json

```diff
@@ -13,7 +13,9 @@
     "clean:all": "gulp clean:all",
     "reinstall": "rimraf node_modules && npm --force cache clean && npm install",
     "test:unit": "gulp test:unit",
-    "semantic-release": "semantic-release pre && gulp build && cp -r .git dist && npm publish dist/ && semantic-release post"
+    "semantic-release": "semantic-release pre && gulp build && cp -r .git dist && npm publish dist/ && semantic-release post",
+    "build-planner": "npm install && npm run build && npm pack dist/",
+    "unit-test": "npm run test:unit"
```

Comment:
What is the difference between `npm run test:unit` and `npm run unit-test`? 
They do the same thing but you move away from the standard format of listing the action first. What is the point?